docs/helm-chart/index.rst

```diff
@@ -122,10 +122,10 @@ The command removes all the Kubernetes components associated with the chart and
 .. note::
   Some kubernetes resources created by the chart `helm hooks <https://helm.sh/docs/topics/charts_hooks/#hook-resources-are-not-managed-with-corresponding-releases>`__ might be left in the namespace after executing ``helm uninstall``, for example, ``brokerUrlSecret`` or ``fernetKeySecret``.
 
-Installing the Chart with Argo CD, Flux or Terraform
+Installing the Chart with Argo CD, Flux, Rancher or Terraform
 -----------------------------------------------------
```

Comment:
Thanks for the contribution.
Please adjust the lengths of line 126 accordingly. Other the doc build may fail.